phpBB/adm/style/acp_profile.html

```diff
@@ -17,7 +17,7 @@
 		</div>
 	<!-- ENDIF -->
 
-	<form id="add_profile_field" method="post" action="{U_ACTION}">
+	<form id="add_profile_field" method="post" action="{U_ACTION}"{S_FORM_ENCTYPE}>
```

Comment:
Not if used as in other places, see links above. The space is supposed to be included in the template var. But it may be safer if included again here...